partcad-cli/src/partcad_cli/click/commands/adhoc/convert.py

```
@@ -0,0 +1,71 @@
+            "gltf",
+        ]),
+    help="Output file type. Inferred from filename if not provided.",
+    required=False,
+)
+@click.argument("input_filename", type=click.Path(exists=True))
+@click.argument("output_filename", type=click.Path(), required=False)
+def cli(input_type, output_type, input_filename, output_filename):
+    """
+    Convert CAD files from one format to another.
+    """
+    def infer_type_from_filename(filename):
```

Ruff (0.8.2)

Missing return type annotation for private function `infer_type_from_filename`

(ANN202)